### DISCUSSION TITLE
Sitecore Author Toolbox on 10.2

### DISCUSSION THREAD

**Discussion by cinter** [ugo-quaisse/sitecore-author-toolbox#42] [General]
Hi @ugo-quaisse !
Could you give a quick update on your plans for troubleshooting the extension on Sitecore 10.2? It works just well enough for us to see how awesome it might be, but then it starts blocking all content with console errors all too soon. Official issue is entered in the Issues tab.
I bet if you gave us a link, you'd get a certain amount of financial support to get this working.

K

**Comment by ugo-quaisse** [comment]
Hello,

I replied to your ticket. I was able to reproduce and fix the issue
locally. I'll run few other test and push a hotfix release.

in the meantime, I might have a workaround, can you check the comment here:
https://github.com/ugo-quaisse/sitecore-author-toolbox/issues/41#issuecomment-1176096794

Let me know if it worked
Ugo

**Reply by cinter** [reply]
@ugo-quaisse It worked! With version 7.1.2, Everything seems to work in 10.2 as a normal author / user without admin privileges. 

Thank you!
